images/postgres/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "tc_postgres"
+version = "0.1.0"
+authors = [ "CoBloX developers <team@coblox.tech>" ]
```

Comment:
Not sure this is valid as we don't use postgres (do we ?)

Reply:
Well the field doesn't say `users` but `authors` :)

Unless what you want to say is: I don't want to maintain this!

Reply:
I am saying I don't think CoBloX should maintain this.

Reply:
Fair enough. I'll add my name then :)

Reply:
Done.